documentation/cli.md

```diff
@@ -264,11 +264,11 @@ It gets trigerred again the next time the bandwidth bypass this threshold.
 
 {% highlight shell %}
 skydive client alert create \
-  --expression "Gremlin(\"G.Flows().Has('Network.A', '192.168.0.1').Metrics().Sum()\").ABBytes > 1*1024*1024" \
+  --expression "JSON.parse(Gremlin(\"G.Flows().Has('Network.A', '192.168.0.1').Metrics().Sum()\")).ABBytes > 1*1024*1024" \
```

Comment:
Thanks for reporting this :) could you just update/fix the expression line 257 as well

```
states = JSON.parse(Gremlin(\"G.V().Has('Name', 'vm1-eth0').Values('State')\")); result = false; for (var i = 0; i != states.length; i++){ if (states[i] == 'DOWN'){ result = true; break;}} result;
```